src/ai_hawk/llm/llm_manager.py

```diff
@@ -730,8 +730,8 @@ def is_job_suitable(self):
         )
         output = self._clean_llm_output(raw_output)
         logger.debug(f"Job suitability output: {output}")
-        score = re.search(r"Score: (\d+)", output).group(1)
-        reasoning = re.search(r"Reasoning: (.+)", output, re.DOTALL).group(1)
+        score = re.search(r"Score:\s*(\d+)", output, re.IGNORECASE).group(1)
```

Comment:
can you add null check before group called.

in case if score / reason didn't come in llm output, lets fall back to default apply (as this is less harmful than default skip)

Reply:
Thats good idea, I've addressed that. So far, responses are parsed correctly, but if we see the warning in applying to jobs with uncertain suitability multiple times, we should look into it further. But this should definitely fix it.